torch/utils/tensorboard/writer.py

```diff
@@ -352,6 +400,31 @@ def add_image(self, tag, img_tensor, global_step=None, walltime=None, dataformat
             convert a batch of tensor into 3xHxW format or call ``add_images`` and let us do the job.
             Tensor with :math:`(1, H, W)`, :math:`(H, W)`, :math:`(H, W, 3)` is also suitible as long as
             corresponding ``dataformats`` argument is passed. e.g. CHW, HWC, HW.
+
+        Examples::
+
+            from torch.utils.tensorboard import SummaryWriter
+            import numpy as np
+            my_img = np.zeros((3, 100, 100))
```

Comment:
Nit: Can we change variable names from `my_img` to `img` and from `my_img_HWC` to `img_hwc`?

Reply:
also fixed similar issue in `add_images`